### PR TITLE
fix: center layout and styled secondary nav

### PR DESCRIPTION
**Pull request by burakukula**
in the next pr:
 - highlight nav on scroll
 - styles for active and hover state for secondary nav

### CODE REVIEW

File: packages/website/components/GlobalStyles.tsx [hunk 1]
```diff
@@ -48,6 +48,20 @@ export const GlobalStyles = () => {
...
+          font-family: ${tokens.fontStackPrimary};
+          font-size: ${tokens.fontSizeL} !important;
+          line-height: ${tokens.lineHeightL} !important;
+        }
+
+        h2 {
+          font-family: ${tokens.fontStackPrimary};
+          font-size: ${tokens.fontSizeL} !important;
+          line-height: ${tokens.lineHeightL} !important;
+        }
```
Comment (burakukula) [inline packages/website/components/GlobalStyles.tsx:63]: We would probably need to figure out how to overwrite other typography styles accordingly

File: packages/website/components/LiveEditor/ComponentSource.tsx [hunk 2]
```diff
@@ -47,6 +47,7 @@ const styles = {
     font-family: ${tokens.fontStackMonospace};
     background-color: #222031;
     color: #ffffff;
+    min-height: 100px;
```
Comment (burakukula) [inline packages/website/components/LiveEditor/ComponentSource.tsx:50]: When not set buttons overlap one line content

File: packages/website/components/LiveEditor/ComponentSource.tsx [hunk 3]
```diff
@@ -59,13 +60,13 @@ const styles = {
   `,
   copyButton: css`
     position: absolute;
-    top: ${tokens.spacingS};
+    bottom: ${tokens.spacingS};
```
Comment (burakukula) [inline packages/website/components/LiveEditor/ComponentSource.tsx:63]: moved it to the bottom, to not overlap the content (at the bottom most likely we have closing tags, so less probably that buttons will cover the content)

File: packages/website/components/LiveEditor/ComponentSource.tsx [hunk 4]
```diff
@@ -95,7 +96,11 @@ export function ComponentSource({ children }: { children: string }) {
           <>
             <LiveError className={styles.error} />
             <div style={{ position: 'relative' }}>
-              <CopyButton className={styles.copyButton} value={children} />
+              <CopyButton
+                tooltipProps={{ placement: 'left' }}
```
Comment (burakukula) [inline packages/website/components/LiveEditor/ComponentSource.tsx:100]: tooltip was shown belove the playground button

File: packages/website/components/GlobalStyles.tsx [hunk 5]
```diff
@@ -48,6 +48,20 @@ export const GlobalStyles = () => {
           margin: 0;
         }
 
+        p,
+        a,
+        li {
+          font-family: ${tokens.fontStackPrimary};
```
Comment (suevalov) [inline packages/website/components/GlobalStyles.tsx:54]: if we want these styles to be applied only for MDX pages, it's better to apply re-styling in `MdxRenderer` components map.

Right now, these overrides affect how our components look like.

**Review by suevalov — CHANGES_REQUESTED** [review]

**Review by suevalov — APPROVED** [review]

**Comment by domarku** [comment]
Looks good! Let's increase the width of the main content container to `50rem` (800px)

**Pull request merged**